internal/docs/sidebar/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import styled from 'styled-components'
+import containerStyles from '../../../core/components/_helpers/container-styles'
```

Comment:
Explained in the other comment

```suggestion
import containerStyles from '@auth0/cosmos/_helpers/container-styles'
```